sno/working_copy/postgis.py

```diff
@@ -333,9 +334,10 @@ def _create_spatial_index(self, sess, dataset):
         # Create the PostGIS Spatial Index
         L.debug("Creating spatial index for %s.%s", dataset.table_name, geom_col)
         t0 = time.monotonic()
+        index_name = f"{dataset.table_name}_idx_{geom_col}"
         sess.execute(
             f"""
-            CREATE INDEX "{dataset.table_name}_idx_{geom_col}"
+            CREATE INDEX {self.quote(index_name)}
```

Comment:
Can we do this using [`Index()`](https://docs.sqlalchemy.org/en/14/dialects/postgresql.html#index-types)?

Reply:
Done